osu.Game/Overlays/Profile/Sections/PaginatedContainer.cs

```diff
@@ -73,28 +76,32 @@ protected PaginatedContainer(Bindable<User> user, string header, string missing)
             };
         }
 
-        [BackgroundDependencyLoader]
-        private void load(RulesetStore rulesets)
+        protected override void LoadComplete()
         {
-            Rulesets = rulesets;
-
-            User.ValueChanged += onUserChanged;
-            User.TriggerChange();
+            base.LoadComplete();
+            User.BindValueChanged(onUserChanged, true);
         }
 
-        private void onUserChanged(ValueChangedEvent<User> e)
+        protected void ClearContent()
         {
             loadCancellation?.Cancel();
             retrievalRequest?.Cancel();
 
             VisiblePages = 0;
             ItemsContainer.Clear();
 
+            moreButton.IsLoading = true;
+        }
+
+        private void onUserChanged(ValueChangedEvent<User> e)
+        {
+            ClearContent();
```

Comment:
Both usages of `ClearContent` also call `ShowMore`. why can't ShowMore be kept private if so?

Reply:
`ClearContent` doesn't call `ShowMore`. And we need `ShowMore` function to use it in `PaginatedScoreContainer`

Reply:
Okay, I see how can it be simplified